03-other-topics/005-k8s-basics.md

```diff
@@ -0,0 +1,81 @@
+---
+category: cloud-platform
+expires: 2019-01-01
+---
+# Kubernetes Basics - Links to helpful beginner resources
+
+The purpose of this page is to serve as a small catalogue of links to basic Kubernetes concepts and learning resources.
+
+## The Concept
+
+Kubernetes is very often referred to as **"K8s"**, purely for the reason, that typing Kubernetes over and over gets quite taxing.
+
+If Kubernetes is a totally new concept to you, then firstly you should probably check out a few of the resources linked below:
+
+Official Website: 
+https://kubernetes.io/
+
+Offical Docs: 
+https://kubernetes.io/docs/concepts/
+
+Quick concept video: 
+https://www.youtube.com/watch?v=IMOZCDhH7do
+
+## Usage Platforms
+
+Kubernetes is normally used with one of the big 3 cloud providers. Amazon Web Services, Google Cloud Engine, and Microsoft Azure.
+
+All 3 have their own documentation and implementation / hosting tools:
+
+### AWS
+
+Kubernetes Docs: 
+https://kubernetes.io/docs/setup/turnkey/aws/
+
+Implementation Tool:
+https://aws.amazon.com/eks/ 
+
+### GCE
+
+Kubernetes Docs: 
+https://kubernetes.io/docs/setup/turnkey/gce/
+
+Implementation Tool:
+https://cloud.google.com/kubernetes-engine/
+
+### Local Hosting
```

Comment:
Should this be a separate section? This is possible something useful as they can experiment themselves on a local installation but it won't be cloud-platform. Is this beginner material?

Reply:
Happy to make it a separate section or take it out altogether. Personally thought it would be worth mentioning if it's the first thing they'll go to when experimenting for the first time.